client/src/app/site/deployment-center/deployment-center-setup/wizard-logic/azure-devops.service.ts

```diff
@@ -97,7 +97,10 @@ export class AzureDevOpsService implements OnDestroy {
     });
   }
 
-  startDeployment(account: string, deploymentObj: any) {
+  startDeployment(account: string, deploymentObj: any, isNewVsoAccount: boolean) {
```

Comment:
Modified the signature of the function since I don't see it being used anywhere else apart from deployment-center-state-manager.ts
Other option is method overloading